mvpa2/mappers/projection.py

```diff
@@ -69,7 +71,8 @@ def __init__(self, demean=True, **kwargs):
         """Offset (most often just mean) in the input space"""
         self._offset_out = None
         """Offset (most often just mean) in the output space"""
-
+        self.ca.pass_fa = None
+        """Feature attribute to be passed onto output dataset"""
```

Comment:
oh hoh... hold on... now that I actually read your test which shows the use case.  This is an abuse of .ca which are intended for storage of computational artifacts, thus not to be assigned outside.

What you really want is just to assign that fa in your postproc which you could give to any node. e.g. you could say

```python
def myawkwardcustomfaassignment(ds):
  ds.fa['magicalone'] = ['whatever']*ds.nfeatures
  return ds
```
and then give  to your e.g. SVDMapper   postproc=myawkwardcustomfaassignment

or am I missing the point?